.github/workflows/release.yml

```diff
@@ -23,6 +23,6 @@ jobs:
       # Required by cosign keyless signing
       id-token: write
 
-    uses: kubewarden/github-actions/.github/workflows/reusable-release-policy-go.yml@v1
+    uses: kubewarden/github-actions/.github/workflows/reusable-release-policy-go.yml@v2
```

Comment:
```suggestion
    uses: kubewarden/github-actions/.github/workflows/reusable-release-policy-go.yml@v2.0.2
```

Reply:
done in https://github.com/kubewarden/hostpaths-psp-policy/pull/14/commits/1fe372d56cf24a2234f517c565170aee49363ae9.